pkg/osquery/table/touchid_system_darwin.go

```diff
@@ -0,0 +1,88 @@
+package table
+
+import (
+	"bytes"
+	"context"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/pkg/errors"
+
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+)
+
+func TouchIDSystemConfig(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
+	t := &touchIDSystemConfigTable{
+		client: client,
+		logger: logger,
+	}
+	columns := []table.ColumnDefinition{
+		table.IntegerColumn("touchid_compatible"),
+		table.TextColumn("secure_enclave_cpu"),
+		table.IntegerColumn("touchid_enabled"),
+		table.IntegerColumn("touchid_unlock"),
+	}
+
+	return table.NewPlugin("kolide_touchid_system_config", columns, t.generate)
+}
+
+type touchIDSystemConfigTable struct {
+	client *osquery.ExtensionManagerClient
+	logger log.Logger
+	config *touchIDSystemConfig
+}
+
+type touchIDSystemConfig struct {
+	touchIDCompatible int
+	secureEnclaveCPU  string
+	touchIDEnabled    int
+	touchIDUnlock     int
+}
+
+// TouchIDSystemConfigGenerate will be called whenever the table is queried.
+func (t *touchIDSystemConfigTable) generate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
+	var results []map[string]string
+	var touchIDCompatible, secureEnclaveCPU, touchIDEnabled, touchIDUnlock string
+
+	// Read the security chip from system_profiler
+	var stdout bytes.Buffer
+	cmd := exec.Command("/usr/sbin/system_profiler", "SPiBridgeDataType")
```

Comment:
Consider using `-xml` and parsing that. I think there's more info

Reply:
I saw this flag and have considered this, but I'm just more familiar with regex than parsing XML in Go so I went this route. The only other info is the firmware version for the chip, which I believe (could be wrong) is directly tied to the macOS version so that information can be inferred that way.